pkg/apis/eventing/v1/trigger_validation.go

```diff
@@ -223,7 +223,7 @@ func ValidateSubscriptionAPIFilter(ctx context.Context, filter *SubscriptionsAPI
 	).Also(
 		ValidateSubscriptionAPIFilter(ctx, filter.Not).ViaField("not"),
 	).Also(
-		ValidateCESQLExpression(ctx, filter.SQL).ViaField("sql"),
+		ValidateCESQLExpression(ctx, filter.CESQL).ViaField("sql"),
```

Comment:
shouldn't this also be `cesql` since that is the API field name?

```suggestion
		ValidateCESQLExpression(ctx, filter.CESQL).ViaField("cesql"),
```